src/Stripe.net/Infrastructure/StringOrObject.cs

```diff
@@ -22,7 +22,7 @@ public static void Map(object value, Action<string> updateId, Action<T> updateOb
             else if (value is string)
             {
                 updateId((string)value);
-                updateObject(null);
+                updateObject(default(T));
```

Comment:
Can you put a real comment on this line?

Reply:
👍